keps/sig-scheduling/1923-prefer-nominated-node/README.md

```diff
@@ -163,19 +163,20 @@ _This section must be completed when targeting alpha to a release._
 _This section must be completed when targeting beta graduation to a release._
 
 * **How can a rollout fail? Can it impact already running workloads?**
-  Try to be as paranoid as possible - e.g., what if some components will restart
-   mid-rollout?
+  No. It's expected to accelerate scheduling a pod upon preemption, and hence it won't impact
+  already running workloads.
 
 * **What specific metrics should inform a rollback?**
+  Noticeable and sustainable increase in `scheduler_framework_extension_point_duration_seconds{extension_point="Filter"}`
+  latency metric.
 
 * **Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?**
   Describe manual testing that was done and the outcomes.
-  Longer term, we may want to require automated upgrade/rollback tests, but we
-  are missing a bunch of machinery and tooling and can't do that now.
+  Manually tested successfully.
 
 * **Is the rollout accompanied by any deprecations and/or removals of features, APIs,
 fields of API types, flags, etc.?**
-  Even if applying deprecation policies, they may still surprise some users.
+  No.
```

Comment:
There are a bunch of answers still missing (e.g. the whole Monitoring Requirements) section.

Please fill in all questions and ping me then.

Reply:
Done